index.html

```diff
@@ -5,38 +5,56 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
-      rel="stylesheet"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
-      rel="stylesheet"
-    />
     <title>Profile Rapheal Okonta</title>
   </head>
   <body>
     <header>
+     
       <nav class="nav_bar">
-        <a href="#" class="name_logo">RAPHEAL OKONTA</a>
-        <div class="link_header">
-        <div class="nav_header_link">
-          <a class="header_menu" href="#">Portfolio</a>
-          <a class="header_menu" href="#">About</a>
-          <a class="header_menu" href="#">Contact</a>
```

Comment:
![contact](https://user-images.githubusercontent.com/93039359/196889981-bbfbbc29-0f7b-4d2c-bbf2-4f1f14e77ad5.PNG)

**[Optional]**

- [ ] Your contact link should have capital letter at the start. I means your **c** should be capital as you can see in the image above. 👍